content/enterprise/release/v202112-2.mdx

```diff
@@ -0,0 +1,41 @@
+---
+page_title: Releases - Terraform Enterprise
+---
+
+# TFE Release v202112-2 (590)
+
+APPLICATION LEVEL BUG FIXES SINCE v202112-1:
+
+1. Updated the version of the terraform-registry that shipped with the v202112-1 release.
+
+UPCOMING DEPRECATION NOTICE:
+
+1. Effective April, 2022 there will be an update to Terraform Enterprise container names. This change may break container monitoring or custom tooling that identifies containers by name. More specific information regarding name changes will be made available in future release notes.
+
+
+APPLICATION LEVEL BREAKING CHANGES:
```

Comment:
do we think it might make sense to put "none" here? I just think having a heading with nothing underneath might look weird.

Reply:
Yeah that's an excellent call. Also wonder if we should change these from numbered lists to bulleted because 

1. None

Feels really funny but having it without breaks the type hierarchy a bit.

Reply:
yes I totally understand! In this ONE instance, I might say you need neither. None could just be on its own without a bullet. Typically when there is only one item, I think it's better to not use bullets.